docs-starlight/src/data/commands/info/print.mdx

```diff
@@ -13,5 +15,18 @@ experiment:
 examples:
   - description: Print out information about the current context of Terragrunt.
     code: |
-      terragrunt info print
+      $ terragrunt info print
+      {
+        "config_path": "/example/path/terragrunt.hcl",
+        "download_dir": "/example/path/.terragrunt-cache",
+        "iam_role": "",
+        "terraform_binary": "tofu",
+        "terraform_command": "print",
+        "working_dir": "/example/path"
+      }
+
 ---
+
+This command outputs a JSON object that includes useful contextual information such as the path to the configuration file,
```

Comment:
Don't duplicate this. It'll show up in the CLI:
https://terragrunt-docs-hyahh7xyq-gruntwork.vercel.app/docs/reference/cli/commands/info/print/